projects/go-lib/src/lib/components/go-select/go-select.component.ts

```diff
@@ -35,6 +35,7 @@ export class GoSelectComponent implements OnInit {
   @Input() placeholder: string;
   @Input() searchable: boolean = true;
   @Input() showSelectAll: boolean = true;
+  @Input() showRemoveAll: boolean = false;
```

Comment:
As stated in my other comment, I don't believe we need this. If you wanted to keep it to be able to override showing/hiding the button, then this should default to `true` and we can keep the binding in the conditional.

Reply:
Hi @grahamhency Done with the changes. Please check